VS_Meadow_Extension/VS_Meadow_Extension.Shared/MeadowDeployProvider.cs

```diff
@@ -143,9 +150,9 @@ private async Task<string> GetOutputPathAsync(string filename)
             return outputPath;
         }
 
-        private static void MeadowConnection_DeviceMessageReceived(object sender, (string message, string source) e)
+        private static async void MeadowConnection_DeviceMessageReceived(object sender, (string message, string source) e)
```

Comment:
this change doesn't really matter since we're not doing any work after the call to ReportDeviceMessage ... but no harm either way